docs/adrs/003-standardizing-locale.md

```diff
@@ -0,0 +1,172 @@
+# ADR 003: Standardizing Locale
+
+Decision: keep the environment empty and don't set anything related to locale.
+This will result in most builds using the `C` locale.
+If someone needs a specific locale, they can set it themselves and their tools will behave like they expect, since we won't do anything to interfere with that.
```

Comment:
alternative: set `LC_ALL=C` and allow it to be overridden. Not as good IMO because it requires writing more code, but we don't see additional benefit by setting it explicitly.